src/app/categories/categories.component.scss

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
not merge blocking , but please remind to indent your files before submit , 
may be you should add an extension  to vscode, so remind me tomorrow